react-native-libraries.json

```
@@ -11942,5 +11942,42 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/mehuljetani/react-native-typing-animated-text",
+    "npmPkg": "react-native-typing-animated-text",
+    "examples": [
+      "https://github.com/mehuljetani/react-native-typing-animated-text"
+    ],
+    "images": [
+      "https://github.com/mehuljetani/react-native-typing-animated-text/blob/main/src/assets/demo.gif"
+    ],
+    "ios": true,
+    "android": true
+  },
+  {
+    "githubUrl": "https://github.com/mehuljetani/react-native-custom-animated-accordion",
+    "npmPkg": "react-native-custom-animated-accordion",
+    "examples": [
+      "https://github.com/mehuljetani/react-native-custom-animated-accordion"
+    ],
+    "images": [
+      "https://github.com/mehuljetani/react-native-custom-animated-accordion/blob/main/src/assets/accord.gif"
+    ],
+    "ios": true,
+    "android": true
+  },
+  {
+    "githubUrl": "https://github.com/mehuljetani/react-native-animated-flip-card",
+    "npmPkg": "react-native-animated-flip-card",
```

Hey @mehuljetani, thank you for adding new entries to the directory! 👍 

```suggestion
    "examples": [
      "https://github.com/mehuljetani/react-native-typing-animated-text"
    ],
    "images": [
      "https://github.com/mehuljetani/react-native-typing-animated-text/blob/main/src/assets/demo.gif"
    ],
    "ios": true,
    "android": true
  },
  {
    "githubUrl": "https://github.com/mehuljetani/react-native-custom-animated-accordion",
    "examples": [
      "https://github.com/mehuljetani/react-native-custom-animated-accordion"
    ],
    "images": [
      "https://github.com/mehuljetani/react-native-custom-animated-accordion/blob/main/src/assets/accord.gif"
    ],
    "ios": true,
    "android": true
  },
  {
    "githubUrl": "https://github.com/mehuljetani/react-native-animated-flip-card",
```

Just small nit, `npmPkg` can be skipped for non-scoped packages, if repository name matches the package name:
* https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg

---

@Simek thanks 👍